<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reçete</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>nenra.ico</normaloff>nenra.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 251, 202);</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">;</string>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>KOD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AÇIKLAMA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>BİRİM</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>N</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>471</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>KOD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AÇIKLAMA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>BİRİM</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MİKTAR</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../muhasebe-2/Pictures/Adsız.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>İptal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background =rgb(255, 16, 60)</string>
   </property>
   <property name="text">
    <string>Satır Sil</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
